ui/src/components/getRun/NameValueTable.jsx

```diff
@@ -0,0 +1,60 @@
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+import { makeStyles } from '@material-ui/core/styles';
+
+import LoadingShimmer from './LoadingShimmer';
+
+const useStyles = makeStyles(_ => ({
+  table: {
+    '& > .MuiTableCell-root': {
+      borderBottom: 'none',
+    },
+  },
+  row: {
+    '& > td': {
+      borderBottom: 'none',
+      fontSize: '18px',
+      lineHeight: '24px',
+    },
+    '& > td:first-child': {
+      color: '#707070',
+    },
+  },
+}));
+
+export const makeRow = (name, value) => ({
+  name,
+  value,
+});
+
+export const NameValueTable = ({ rows, rowsToLoad }) => {
+  const classes = useStyles();
+
+  let content;
+  if (rows) {
+    content = rows.map(row => (
```

Comment:
incidentally, I think a tuple would be cleaner and reduce the need for `makeRow`.
```suggestion
    content = rows.map(([name, value])=> (
```

Reply:
Done